dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -237,22 +200,20 @@ def writeErrors(self, failures, interfaces, job_id, short_colnames, writer, warn
             elif failure[4] == "warning":
                 # write to warnings file
                 warning_writer.write([field_name,error_msg,str(row_number),failed_value,original_rule_label])
-            error_list.recordRowError(job_id,self.filename,field_name,error,row_number,original_rule_label,severity_id=severityId)
+            error_list.recordRowError(job_id,job.filename,field_name,error,row_number,original_rule_label,severity_id=severityId)
         return fatal_error_found
 
-    def runValidation(self, job_id, interfaces):
+    def runValidation(self, job, interfaces):
```

Comment:
So much love for passing around models instead of querying based on ids for everything.